docs/mp/security/03_configuration-secrets.adoc

```diff
@@ -91,9 +91,11 @@ The config encryption filter provides a Main class `io.helidon.config.encryption
 [source,bash]
 .Encrypt secret `secretToEncrypt` using shared secret `masterPassword`
 ----
-java io.helidon.config.encryption.Main aes masterPassword secretToEncrypt
+java -jar helidon-config-encryption-X.X.X.jar aes masterPassword secretToEncrypt
 ----
```

Comment:
Our `.adoc` documents can use attribute `helidon-version` so you can replace `X.X.X` with `{helidon-version}`, here and in later lines in the same file.

Reply:
done!